src/runner/index.js

```diff
@@ -123,16 +123,28 @@ export default class Runner extends EventEmitter {
     _validateRunOptions () {
         const concurrency = this.bootstrapper.concurrency;
         const speed       = this.opts.speed;
-        const proxyBypass = this.opts.proxyBypass;
+        let proxyBypass   = this.opts.proxyBypass;
 
         if (typeof speed !== 'number' || isNaN(speed) || speed < 0.01 || speed > 1)
             throw new GeneralError(MESSAGE.invalidSpeedValue);
 
         if (typeof concurrency !== 'number' || isNaN(concurrency) || concurrency < 1)
             throw new GeneralError(MESSAGE.invalidConcurrencyFactor);
 
-        if (proxyBypass)
-            assertType(is.string, null, '"proxyBypass" argument', proxyBypass);
+        if (proxyBypass) {
+            assertType([ is.string, is.array ], null, '"proxyBypass" argument', proxyBypass);
+
+            if (typeof proxyBypass === 'string')
+                proxyBypass = [ proxyBypass ];
```

Comment:
We don't add additional spaces for separating array contents and square brackets. `[proxyByass]` is fine.